src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -206,6 +206,13 @@ static std::map<qr_type, query_entry> query_table = {
 
   { qr_type::QR_POWER_MICROWATTS,          {std::bind(sysfs_user, sp::_1, sp::_2, sp::_3, "xmc", "xmc_power")}},
 
+  // { qr_type::QR_MIG_ECC_ENABLED,           {std::bind(sysfs_user, sp::_1, sp::_2, sp::_3, sp::_4, "ecc_enabled")}},
+  // { qr_type::QR_MIG_ECC_STATUS,            {std::bind(sysfs_user, sp::_1, sp::_2, sp::_3, sp::_4, "ecc_status")}},
+  // { qr_type::QR_MIG_ECC_CE_CNT,            {std::bind(sysfs_user, sp::_1, sp::_2, sp::_3, sp::_4, "ecc_ce_cnt")}},
+  // { qr_type::QR_MIG_ECC_UE_CNT,            {std::bind(sysfs_user, sp::_1, sp::_2, sp::_3, sp::_4, "ecc_ue_cnt")}},
+  // { qr_type::QR_MIG_ECC_CE_FFA,            {std::bind(sysfs_user, sp::_1, sp::_2, sp::_3, sp::_4, "ecc_ce_ffa")}},
+  // { qr_type::QR_MIG_ECC_UE_FFA,            {std::bind(sysfs_user, sp::_1, sp::_2, sp::_3, sp::_4, "ecc_ue_ffa")}},
+
```

Comment:
@stsoe I need the user to pass in a 4th argument which specifies the tag, i.e., bank0, bank1 etc. Can you help me figure out how to achieve that? 
Thanks.